### PR TITLE
chore(#9327): adjust new nav styles after manual testing

### PR DESCRIPTION
**Pull request by Benmuiruri**
# Description
 
<details>
<summary>selected icon and label have color #FFFFFF</summary>

Screenshot 2024-09-04 at 15 13 38

</details>

<details>
<summary>Unselected icon and label have color #A7A9AC </summary>

Screenshot 2024-09-04 at 15 13 20

</details>

<details>
<summary>Snackbar is above FAB and adjusts when Tab navigation is not present</summary>

https://github.com/user-attachments/assets/3207e6d7-3ad9-4b76-af6a-66d3c890a901

</details>

<details>
<summary>Removed the white outline for the notification dot </summary>

Screenshot 2024-09-04 at 15 23 24

</details>

<details>
<summary>Made the highlight pill consistent size </summary>

Screenshot 2024-09-04 at 15 19 55

</details>

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:move-nav-feedback/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:move-nav-feedback/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:move-nav-feedback/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: webapp/src/css/inbox.less [hunk 1]
```diff
@@ -248,10 +249,12 @@ body {
       }
 
       &.selected {
+        text-shadow: none;
         .button-label,
         .mm-icon:hover,
```
Comment (latin-panda) [inline webapp/src/css/inbox.less:254]: To have the pills of the same size, add weight and height to .mm-icon
So when the tab stretches, the pill stays the same size

**Review by latin-panda — CHANGES_REQUESTED** [review]
Nice job!! Just some minor suggestions

File: webapp/src/css/inbox.less (hunk 1 above)
Comment (Benmuiruri) [reply webapp/src/css/inbox.less:256]: Fixed it by adding the `width` and `height` to the `mm-icon` class not only when it's selected

**Comment by Benmuiruri** [comment]
Hi @tatilepizs with Jenni out for a couple of days, please review this to check changes after feedback. Thanks

**Review by tatilepizs — APPROVED** [review]
I reviewed the changes mentioned in the description and the received feedback and everything looks good, thanks Ben! ⭐ 

Just one thing, I remembered about [this comment](https://github.com/medic/cht-core/pull/9357#discussion_r1740665572) that Jenny added in a previous PR, so I checked all the different tabs and you can still see the problem in `Tasks`. Not sure if it should be fixed in this PR or in a new one, that is why I am approving it because everything related to the styles looks good 😊 

[video](https://github.com/user-attachments/assets/5ecdeedc-72cb-4096-9574-c62fc01fbbb4)

File: webapp/src/css/inbox.less [hunk 2]
```diff
@@ -627,7 +632,7 @@ body {
 }
 
 .loading-status {
-  margin: 10px 10px @bottom-navbar-height;
+  margin: 10px 10px calc(@bottom-navbar-height + 10px);
```
Comment (latin-panda) [inline webapp/src/css/inbox.less:635]: We had to add a bottom margin in some pages, are they all the same 10 to 15px? Which can be a good variable
`@navbar-page-margin`, and extract these to 1 rule. For example:

```
.page-1 .loading-status
.page-2 .list-component
.page-X ..... { <---- next time we need to add this margin, we just add the page and the element to attach the margin
  margin-bottom: calc(@bottom-navbar-height + @navbar-page-margin);
}
```

**Pull request merged**